torch/tensor.py

```diff
@@ -391,12 +413,12 @@ def __rfloordiv__(self, other):
 
     __neg__ = _C._TensorBase.neg
 
-    __eq__ = _C._TensorBase.eq
-    __ne__ = _C._TensorBase.ne
-    __lt__ = _C._TensorBase.lt
-    __le__ = _C._TensorBase.le
-    __gt__ = _C._TensorBase.gt
-    __ge__ = _C._TensorBase.ge
+    __eq__ = _wrap_type_error_to_not_implemented(_C._TensorBase.eq)
```

Comment:
Can you remind me why we don't implement the wrapping here directly in C++?

Reply:
It doesn't make sense to return `NotImplemented` from `Tensor.eq` which is what this is wrapping.